docassemble/AssemblyLine/data/questions/al_visual.yml

```diff
@@ -41,11 +41,15 @@ default screen parts:
   corner back button label: |
     Undo
   pre: |
-    % if get_config('debug') and not url_args.get("hide_hud", "").lower() in ["1", "on", "true", "yes"]:
-    `id: ${ user_info().question_id }`[BR]
-    `Variable: ${ user_info().variable }`[BR]
-    `Version: ${ user_info().package.replace("docassemble.", "") + " " if not user_info().package.startswith("docassemble.playground") else "" }${ package_version_number }; ${ al_version }`
-    <div data-variable="${ encode_name(str( user_info().variable )) }" id="sought_variable" aria-hidden="true" style="display: none;"></div>
+    % if (get_config('debug') and not url_args.get("hide_hud", "").lower() in ["1", "on", "true", "yes"]) or (url_args.get("debug") in ["1", "on", "true", "yes"]):
+    `id: ${ current_context().question_id }`[BR]
+    `Variable: ${ current_context().variable }`[BR]
+    `Version: ${ current_context().package.replace("docassemble.", "") + " " if not current_context().package.startswith("docassemble.playground") else "" }${ package_version_number }; ${ al_version }`[BR]
+    `Session: ${ current_context().session }`
+    % if (url_args.get("debug") in ["1", "on", "true", "yes"]) and user_has_privilege(["admin", "developer", "advocate"]):
```

Comment:
This doesn't test the URL args as the previous logic does. Is that deliberate?

Reply:
it's nested logic.